ambry-protocol/src/main/java/com.github.ambry.protocol/AdminRequest.java

```diff
@@ -44,22 +45,17 @@
    */
   public static AdminRequest readFrom(DataInputStream stream, ClusterMap clusterMap) throws IOException {
     Short versionId = stream.readShort();
-    if (!versionId.equals(ADMIN_REQUEST_VERSION_V1)) {
-      throw new IllegalStateException("Unrecognized version for AdminRequest: " + ADMIN_REQUEST_VERSION_V1);
+    if (!versionId.equals(ADMIN_REQUEST_VERSION_V2)) {
```

Comment:
There is no reason to be backward compatible since the only client is the server admin tool and not a service.